<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciador De Tarefas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_tarefas">
       <attribute name="title">
        <string>Tarefas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="4">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableWidget" name="tarefas_tableWidget">
             <column>
              <property name="text">
               <string>id</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nome Tarefa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Descricao</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prazo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Criacao</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Concluido</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Importante</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Id Projeto</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="criar_button">
            <property name="text">
             <string>Criar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editar_button">
            <property name="text">
             <string>Editar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="excluir_button">
            <property name="text">
             <string>Excluir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importar_button">
            <property name="text">
             <string>Importar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportar_button">
            <property name="text">
             <string>Exportar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="marcar_concluido_button">
            <property name="text">
             <string>Marcar Conlcuido</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="marcar_importante_button">
            <property name="text">
             <string>Marcar Importante</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sair_button">
            <property name="text">
             <string>Sair</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="tarefa_pendente_radio">
            <property name="text">
             <string>Tarefas Pendentes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="tarefa_concluida_radio">
            <property name="text">
             <string>Tarefas Concluidas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="todas_tarefas_radio">
            <property name="text">
             <string>Todas Tarefas</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item alignment="Qt::AlignRight">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Filtrar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="filtrar_box">
             <item>
              <property name="text">
               <string>Todas Tarefas</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tarefas Pendentes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tarefas Concluidas</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_projetos">
       <attribute name="title">
        <string>Projetos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Filtrar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="filtrar_pr_box">
             <item>
              <property name="text">
               <string>Todos Projetos</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Projetos Pendentes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Projetos Concluidos</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QRadioButton" name="projeto_pendente_radio">
            <property name="text">
             <string>Projetos Pendentes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="projeto_concluido_radio">
            <property name="text">
             <string>Projetos Concluidas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="todos_projetos_radio">
            <property name="text">
             <string>Todos Projetos</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTableWidget" name="tableWidget_2">
             <column>
              <property name="text">
               <string>Id</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nome Projeto</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Descricao</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prazo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Criacao</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Concluido</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Importante</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="criar_pr_button">
            <property name="text">
             <string>Criar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editar_pr_button">
            <property name="text">
             <string>Editar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="excluir_pr_button">
            <property name="text">
             <string>Excluir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importar_pr_button">
            <property name="text">
             <string>Importar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportar_pr_button">
            <property name="text">
             <string>Exportar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="marcar_concluido_pr_button">
            <property name="text">
             <string>Marcar Conlcuido</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="marcar_importante_pr_button">
            <property name="text">
             <string>Marcar Importante</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sair_pr_button">
            <property name="text">
             <string>Sair</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
